protocols/dcutr/src/protocol/inbound.rs

```diff
@@ -128,7 +125,7 @@ impl PendingConnect {
 #[derive(Debug, Error)]
 pub enum UpgradeError {
     #[error(transparent)]
-    Codec(#[from] prost_codec::Error),
+    Codec(#[from] quick_protobuf_codec::Error),
```

Comment:
Looks like it is https://github.com/libp2p/rust-libp2p/pull/3214/files#diff-1139510a64e6bc16c649cc904d0c3c799663ee17f7828f2dea6839cfb3f5c283R39.

I see that there are several places where this pattern has been implemented. I suggest we split this into a separate task because it's a chunk of work and I fear I'm gonna have limited bandwidth soon.